src/modules/common/components/Tile.js

```diff
@@ -33,13 +33,22 @@ class Tile extends React.Component<PropsType> {
 
   getTile (): React.Node {
     const tile = this.props.tile
-    return tile.isExternalUrl
-      ? <a href={this.props.tile.path} target='_blank'>
-        {this.getTileContent()}
-      </a>
-      : <Link to={this.props.tile.path}>
+    if (!tile.isExternalUrl) {
+      return <Link to={tile.path}>
         {this.getTileContent()}
       </Link>
+    } else if (!tile.postData) {
+      return <a href={tile.path} target='_blank'>
+        {this.getTileContent()}
+      </a>
+    } else {
+      const inputs = []
+      tile.postData.forEach((value, key) => inputs.unshift(<input type='hidden' value={value} key={key} name={key} />))
+      return <form method='POST' action={tile.path} target='_blank'>
```

Comment:
Is there already an example where this is used or do we have to wait for the cms? Cause klicking on the Lehrstellenradar for Augsburg just opens the same page as before without PLZ entered.